qiskit_ibm_runtime/estimator.py

```diff
@@ -16,8 +16,6 @@
 
 from qiskit.circuit import QuantumCircuit, Parameter
 from qiskit.quantum_info import SparsePauliOp
-
-# TODO import BaseEstimator and EstimatorResult from terra once released
```

Comment:
We still need these TODOS to remember to import BaseEstimator and EstimatorResult from terra. We are using a copied version of base classes from terra for now since we expect more breaking changes in the base classes.